setup/update-pib.sh

```diff
@@ -13,11 +13,19 @@ ROS_WORKING_DIR="$USER_HOME/ros_working_dir"
 DEFAULT_NGINX_DIR="/etc/nginx"
 DEFAULT_NGINX_HTML_DIR="$DEFAULT_NGINX_DIR/html"
 
-CEREBRA_ARCHIVE_URL_PATH="https://pib.rocks/wp-content/uploads/pib_data/cerebra-latest.zip"
-CEREBRA_ARCHIVE_NAME="cerebra-latest.zip"
+FRONTEND_REPO="https://github.com/pib-rocks/cerebra.git"
+BACKEND_REPO="https://github.com/pib-rocks/pib-backend.git"
 
-NGINX_CONF_FILE="nginx.conf"
-NGINX_CONF_FILE_URL="https://raw.githubusercontent.com/pib-rocks/setup-pib/main/setup_files/nginx.conf"
+# Create an associative array (=map). This will be filled with repo-origin branch-name pairs
+declare -A repo_map
+repo_map["$FRONTEND_REPO"]="main"
+repo_map["$BACKEND_REPO"]="main"
```

Comment:
is it really necessary to use an associative array here? we never loop over it and the values are never reassigned. Wouldn't it be simpler to just declare variables "FRONTEND_REPO_BRANCH" and "BACKEND_REPO_BRANCH" (or simply use hardcoded value "main" directly)?